README.md

```diff
@@ -95,6 +95,7 @@ module "my_app" {
 | codedeploy_termination_wait_time | number | the number of minutes to wait after a successful blue/green deployment before terminating instances from the original environment | 15 |
 | codedeploy_test_listener_port | number | The port for a codedeploy test listener. If provided CodeDeploy will use this port for test traffic on the new replacement set during the blue-green deployment process before shifting production traffic to the replacement set | null |
 | codedeploy_lifecycle_hooks | [object](#codedeploy_lifecycle_hooks) | Define Lambda Functions for each CodeDeploy [lifecycle event hooks](https://docs.aws.amazon.com/codedeploy/latest/userguide/reference-appspec-file-structure-hooks.html). Use the Lambda function names as the values. Use null if you don't want to invoke a lambda function at that specific hook. Or set this variable to null to not have any lifecycle hooks invoked | `null` |
+| appspec_filename | string | Filename (including path) to use when outputing appspec json. | ${path.cwd}/appspec.json |
```

Comment:
Not sure if the default value here will confuse anyone. What would happen if someone entered `"${path.cwd}/appspec2.json"` for this variable? Will terraform try to convert that ${path.cwd} variable or will it try to create the file at that location without interpolating? I'm not sure if there's a good way around this possible confusion though

Reply:
Changed to:
`appspec.json` in the current working directory (i.e. where you ran `terraform apply`)